src/Stores/Cases/actions.js

```diff
@@ -0,0 +1,64 @@
+/* @flow */
+import type { Dispatch } from '../../types';
+import type { Case, CaseId } from './types';
+import actionTypes from './actionTypes';
+
+import {
+  getParliamentCaseById as apiGetParliamentCaseById,
+  getParliamentCases as apiGetParliamentCases,
+} from '../../utils/api';
+
+/* 
+    GET CASE BY ID
+*/
+
+const getCaseById = () => ({ type: actionTypes.GET_CASE_BY_ID_REQUEST });
+
+const getCaseByIdSuccess = (parliamentCase: Case) => ({
+  case: parliamentCase,
+  type: actionTypes.GET_CASE_BY_ID_SUCCESS,
+});
+
+const getCaseByIdFailure = (error: *) => ({
```

Comment:
`const getCaseByIdFailure = (error: string) => ({` no?

you shouldn't even need to write a type definition if the function is only used locally

Reply:
very true, I think I might have copied this from somewhere else though :)